tools/asg-hevc/asg-hevc.sln

```diff
@@ -0,0 +1,41 @@
+﻿
```

Comment:
@dmitryermilov : do we need this sln? I would drop it as well.

Reply:
Well, it's controversial. We have separate solution for samples. Following this logic we need to drop them in samples as well.

Reply:
These are not samples. If we will merge them in we will need to support them. Will we? For example, we don't even CI test these solutions: we use on AllBuild.sln

Reply:
Why samples require different approach ?

Reply:
They don't, but their solution files existed before. If I would make a decision, I would delete them all, including individual solutions for samples and tutorials.

Reply:
I see. Makes sense to me.